.ci/linters/c/alloc_linter.R

```diff
@@ -2,8 +2,8 @@
 # More specifically, given that this is an AST-ignorant checker,
 #   1. Find groups of malloc()/calloc() calls
 #   2. Check the next line for a check like 'if (!x || !y)'
-alloc_linter = function(c_file) {
-  lines <- readLines(c_file)
+alloc_linter = function(c_obj) {
+  lines <- c_obj$lines
```

Comment:
I know its not directly `data.table` but still feels strange that we do not use `=` for assignment in this file (style linting the linter files)